actionners/aws/lambda/lambda.go

```diff
@@ -0,0 +1,131 @@
+package lambda
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/Falco-Talon/falco-talon/internal/aws/client"
+	"github.com/Falco-Talon/falco-talon/internal/events"
+	"github.com/Falco-Talon/falco-talon/internal/rules"
+	"github.com/Falco-Talon/falco-talon/utils"
+	"github.com/aws/aws-sdk-go-v2/service/lambda"
+	"github.com/aws/aws-sdk-go-v2/service/lambda/types"
+	"github.com/mitchellh/mapstructure"
+	"strconv"
+)
+
+func Action(action *rules.Action, event *events.Event) (utils.LogLine, error) {
+
+	lambdaClient := client.GetAWSClient().GetLambdaClient()
+	parameters := action.GetParameters()
+
+	lambdaConfig, err := NewLambdaConfig(parameters)
+	if err != nil {
+		return utils.LogLine{
+				Objects: nil,
+				Error:   err.Error(),
+				Status:  "failure",
+			},
+			err
+	}
+
+	objects := map[string]string{
+		"name":    lambdaConfig.AWSLambdaName,
+		"version": lambdaConfig.AWSLambdaAliasOrVersion,
+	}
+
+	payload, err := json.Marshal(event.OutputFields)
+	if err != nil {
+		return utils.LogLine{
+				Objects: objects,
+				Error:   err.Error(),
+				Status:  "failure",
+			},
+			err
+	}
+
+	input := &lambda.InvokeInput{
+		FunctionName:   &lambdaConfig.AWSLambdaName,
+		ClientContext:  nil,
+		InvocationType: getInvocationType(lambdaConfig.AWSLambdaInvocationType),
+		Payload:        payload,
+		Qualifier:      &lambdaConfig.AWSLambdaAliasOrVersion,
+	}
+
+	lambdaOutput, err := lambdaClient.Invoke(context.Background(), input)
+	if err != nil {
+		return utils.LogLine{
+				Objects: objects,
+				Error:   err.Error(),
+				Status:  "failure",
+			},
+			err
+	}
+
+	objects = map[string]string{
+		"name":          lambdaConfig.AWSLambdaName,
+		"version":       lambdaConfig.AWSLambdaAliasOrVersion,
+		"status_code":   strconv.FormatInt(int64(lambdaOutput.StatusCode), 10),
+		"lamba_payload": string(lambdaOutput.Payload),
+	}
+
+	output := fmt.Sprintf("the lambda %v:%v has been executed.", lambdaConfig.AWSLambdaName, lambdaConfig.AWSLambdaAliasOrVersion)
+
+	return utils.LogLine{
+			Objects: objects,
+			Output:  output,
+			Status:  "success",
+		},
+		nil
+}
```

Comment:
```suggestion
	status := "success"
	if strconv.FormatInt(int64(lambdaOutput.StatusCode), 10) != http.StatusOK {
		status = "failure"
	}

	return utils.LogLine{
			Objects: objects,
			Output:  string(lambdaOutput.Payload),
			Status:  "success",
		},
		nil
}
```

See what I did for the `kubernetes:exec`

Reply:
That is definitely cleaner, thank you

Reply:
We don't need to cast the status code here though:
```
	if lambdaOutput.StatusCode != http.StatusOK {
		status = "failure"
	}
```